build_tools/bazel_to_cmake/bazel_to_cmake.py

```diff
@@ -392,18 +392,11 @@ def cc_library(self,
     alwayslink_block = self._convert_alwayslink_block(alwayslink)
     testonly_block = self._convert_testonly_block(testonly)
 
-    self.converter.body += """iree_cc_library(
-%(name_block)s%(hdrs_block)s%(textual_hdrs_block)s%(srcs_block)s%(data_block)s%(deps_block)s%(alwayslink_block)s%(testonly_block)s  PUBLIC
-)\n\n""" % {
-    "name_block": name_block,
-    "hdrs_block": hdrs_block,
-    "textual_hdrs_block": textual_hdrs_block,
-    "srcs_block": srcs_block,
-    "data_block": data_block,
-    "deps_block": deps_block,
-    "alwayslink_block": alwayslink_block,
-    "testonly_block": testonly_block,
-    }
+    self.converter.body += (
```

Comment:
This is a great improvement. I would go even further and put each block on its own line

Reply:
Sure! That would look like this:
```python
   self.converter.body += (
      f"iree_cc_library(\n"
      f"{name_block}"
      f"{hdrs_block}"
      f"{textual_hdrs_block}"
      f"{srcs_block}"
      f"{data_block}"
      f"{deps_block}"
      f"{alwayslink_block}"
      f"{testonly_block}"
      f"  PUBLIC\n)\n\n")
```

Oddly, adding them on each line (e.g. `    + deps_block`) is significantly slower, so that's the motivation for the individual f"{string}" lines.

Reply:
Yup python string addition is slow :-)